Makefile

```diff
@@ -59,6 +59,10 @@ uninstall:
 	$(eval DESTDIR ?= $(GOBIN))
 	rm $(GOBIN)/$(OUTPUT)
 
+# TODO: WE MIGHT WANT TO ADD A TOP-LEVEL mock COMMAND TO GENERATE ALL OF THE MOCK INTERFACES
+mock_houston:
+	mockery --filename=ClientInterface.go --output=airflow/mocks --dir=houston --name ClientInterface
```

Comment:
sorry I didn't realise this earlier, but it would be better to keep mocks of an interface either where interface is defined or where it is being used, so you might want to move this mocks to `houston` folder instead of `airflow`

Reply:
Yeah I agree with you, I just wanted to keep the same conventions already configured in the project.